### PR TITLE
Add linter_check.sh

### PR DESCRIPTION
**Pull request by aboudreault**
A convenient script to ensure our changes will pass on travis.

### CODE REVIEW

File: linter_check.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# linter_check ensures your changes will pass on travis.
+# Requires pep3 and flake3: pip install pep8 flake8
```
Comment (ptnapoleon) [inline linter_check.sh]: s/pep3/pep8

Comment (aboudreault) [inline linter_check.sh]: haha!

**Comment by ptnapoleon** [comment]
I like it. Did not run locally to verify correctness

File: linter_check.sh (hunk 1 above)
Comment (ptnapoleon) [inline linter_check.sh]: Oh also flake3. What keyboard do you have that 3 and 8 are near each other?

Comment (aboudreault) [inline linter_check.sh]: no idea what happened... both keys are not near at all :)

**Pull request merged**